server-common/src/main/java/org/apache/kafka/server/common/TestFeatureVersion.java

```diff
@@ -24,7 +24,7 @@ public enum TestFeatureVersion implements FeatureVersion {
     // TEST_1 released right before MV 3.7-IVO was released, and it has no dependencies
     TEST_1(1, MetadataVersion.IBP_3_7_IV0, Collections.emptyMap()),
     // TEST_2 released right before MV 3.8-IVO was released, and it depends on this metadata version
-    TEST_2(2, MetadataVersion.IBP_3_8_IV0, Collections.singletonMap(MetadataVersion.FEATURE_NAME, MetadataVersion.IBP_3_8_IV0.featureLevel()));
+    TEST_2(2, MetadataVersion.IBP_4_0_IVO, Collections.singletonMap(MetadataVersion.FEATURE_NAME, MetadataVersion.IBP_4_0_IVO.featureLevel()));
```

Comment:
@jolshan : Should we keep the MV for TEST_2 fixed at IBP_3_8_IV0?

Reply:
Yeah, I was thinking about this. If we should always have this version be an unstable version.

I think there are trade offs either way. We get the unstable version test coverage, but we have to keep updating it every time the MV becomes stable.

Reply:
I have a plan for this. I plan to keep the latest version unstable and create a TEST_3. TEST_2 will be used for the dependency test and it can be either 3.8 or 4.0.